src/components/Preview.js

```diff
@@ -101,6 +117,15 @@ class Preview extends Component {
               background: gifProgress === 1 ? '#2ecc40' : '#e79600'
             }}
           />
+          <div className="previewInfoContainer">
```

Comment:
the info-box seems to be a child div of the `.Preview-progress-outer` div, seems this could be in its own div